routing/routes_builder/routes_builder_tool/routes_builder_tool.cpp

```diff
@@ -0,0 +1,116 @@
+#include "routing/routes_builder/routes_builder_tool/utils.hpp"
+
+#include "routing/routing_quality/api/mapbox/mapbox_api.hpp"
+
+#include "routing/routes_builder/routes_builder.hpp"
+
+#include "geometry/latlon.hpp"
+
+#include "base/assert.hpp"
+#include "base/file_name_utils.hpp"
+#include "base/logging.hpp"
+
+#include <exception>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <thread>
+#include <vector>
+
+#include "3party/gflags/src/gflags/gflags.h"
+
+DEFINE_uint64(threads, 0, "The number of threads. Use std::thread::hardware_concurrency() by default.");
+
+DEFINE_string(routes_file, "", "Path to file with routes in format: \n\t"
+                               "first_start_lat first_start_lon first_finish_lat first_finish_lon\n\t"
+                               "second_start_lat second_start_lon second_finish_lat second_finish_lon\n\t"
+                               "...");
+
+DEFINE_string(dump_path, "", "Path were routes will be dumped after building."
+                             "Useful for intermediate results, because routes building "
+                             "is long process.");
```

Comment:
```c++
DEFINE_string(dump_path, "", "Path where routes will be dumped after building."
                             "Useful for intermediate results, because routes building "
                             "is a long process.");
```